clearwater-diags-monitor/usr/share/clearwater/bin/clearwater_diags_monitor

```diff
@@ -139,29 +139,59 @@ move_to_dump()
 # List all the clearwater packages installed.
 clearwater_packages()
 {
-  dpkg-query -W -f='${Package} ${Maintainer}\n' | grep " Project Clearwater Maintainers " | cut -d ' ' -f 1
+  {
+    if which rpm > /dev/null ; then
+      rpm -qa --qf '%{NAME} %{PACKAGER}\n'
```

Comment:
I haven't checked all these rpm invocations as I assume you've done something sensible. Shout if there are any you want a second opinion on.